docs/integrations/builtin/credentials/mongodb.md

```diff
@@ -6,42 +6,57 @@ contentType: integration
+- Create a user account with the appropriate permissions on a [MongoDB](https://www.mongodb.com/){:target=_blank .external-link} server.
+- As a Project Owner, add all the [n8n IP addresses](/choose-n8n/cloud/#cloud-ip-addresses) to the IP Access List Entries in the project's **Network Access**. Refer to [Add IP Access List entries](https://www.mongodb.com/docs/atlas/security/ip-access-list/#add-ip-access-list-entries){:target=_blank .external-link} for detailed instructions.
 
-## Using connection string or values
+If you are setting up MongoDB from scratch, create a cluster and a database. Refer to the [MongoDB Atlas documentation](https://www.mongodb.com/docs/atlas/){:target=_blank .external-link} for more detailed instructions on these steps.
 
-Choose how to connect to MongoDB by selecting the type of connection in **Configuration Type**.
-For formatting details for **Connection String**, refer to [Connection String URI Format](https://www.mongodb.com/docs/manual/reference/connection-string/){:target=_blank .external-link}.
+## Supported authentication methods
 
+- Database connection - Connection string
+- Database connection - Values
 
-## Connecting to MongoDB Atlas
+## Related resources
 
-You need to configure your MongoDB database, then configure your n8n credential.
+Refer to the [MongoDBs Atlas documentation](https://www.mongodb.com/docs/atlas/){:target=_blank .external-link} for more information about the service.
 
-### MongoDB Atlas settings
+## Using database connection - Connection string
 
-1. Inside your MongoDB organization, set up your database cluster. Your cluster will be visible in **Organization > Project**, in the **Database** section.
-![Setting up the cluster in MongoDB Atlas](/_images/integrations/builtin/credentials/mongodb/cluster.png)
-2. Inside the cluster, you need to have a database to connect to. If you don't have a database yet, you can go to **Collections** and select **Add My Own Data**. A dialog will show up: enter the name of the database in the **Database name** field and the name of the collection in the **Collection name** field.
-![The dialog to create a new database in MongoDB Atlas](/_images/integrations/builtin/credentials/mongodb/database_create.png)
-3. Once ready, you'll see the database in **Organization > Project > Databases > Collections**.
-![A database and collection in MongoDB Atlas](/_images/integrations/builtin/credentials/mongodb/collections.png)
-4. In the **Network Access** section of your project you need to allow list the n8n IPs. Select **Add IP Address** button and add all the IPs listed [here](/choose-n8n/cloud/#cloud-ip-addresses). The allow list IPs will be visible in the **IP Access List** table.
-![The list of IPs allowed to connect to the cluster in MongoDB Atlas](/_images/integrations/builtin/credentials/mongodb/network_access.png)
+To configure this credential, you'll need:
 
-### n8n settings
+- Select `Connection String` as the **Configuration Type**.
```

Comment:
As previous comment. I think this should definitely be bold, unless they need to type Values in? But if it's an option they're clicking/selecting in the UI, it should be bold